hardware_applications/week2/starter_main.cpp

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
do we want to give them includes?

Reply:
Up to you. IIRC we're teaching includes this week, so it seemed like a good idea to make them type them out themselves to reinforce the learning they just did.